infrastructure/cdn-in-a-box/cache/run.sh

```diff
@@ -17,14 +17,78 @@
 # specific language governing permissions and limitations
 # under the License.
 
-set -e
-set -x
-set -m
+set -o errexit -o xtrace -o monitor
+
+mkdir /tmp/ort
+
+set-dns.sh
+insert-self-into-dns.sh
+
+source /to-access.sh
+
+# Wait on SSL certificate generation
+until [[ -f "$X509_CA_ENV_FILE" ]]
+do
+  echo "Waiting on Shared SSL certificate generation"
+  sleep 3
+done
+
+# Source the CIAB-CA shared SSL environment
+until [[ -n "$X509_GENERATION_COMPLETE" ]]
+do
+  echo "Waiting on X509 vars to be defined"
+  sleep 1
+  source "$X509_CA_ENV_FILE"
+done
+
+# Trust the CIAB-CA at the System level
+cp "$X509_CA_CERT_FULL_CHAIN_FILE" /etc/pki/ca-trust/source/anchors
+update-ca-trust extract
+
+while ! to-ping 2>/dev/null; do
+	echo "waiting for Traffic Ops"
+	sleep 5
+done
+
+export TO_USER=$TO_ADMIN_USER
+export TO_PASSWORD=$TO_ADMIN_PASSWORD
+
+# wait until the CDN has been registered
+found=
+while [[ -z $found ]]; do
+    echo 'waiting for enroller setup'
+    sleep 3
+    found=$(to-get api/2.0/cdns?name="$CDN_NAME" | jq -r '.response[].name')
+done
 
 for f in /opt/init.d/*; do
     echo "$f"
-    $f
+    source "$f"
+done
+
+# Wait for SSL keys to exist
+until [[ $(to-get "api/2.0/cdns/name/$CDN_NAME/sslkeys" | jq '.response | length') -gt 0 ]]; do
+	echo 'waiting for SSL keys to exist'
+	sleep 3
```

Comment:
Inconsistent indentation using tabs - existing code in this file used 4 spaces. I wouldn't object to switching to tabs, but it should be done consistently and globally through the file

Reply:
Spacing made consistent in 4d2c0da326.